source/default/_patterns/02-molecules/tabs/index.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * tabs
+ */
+
+import $ from 'jquery';
+
+// Module dependencies
+import 'protons';
+
+// Module template
+import './_tabs.twig';
+
+// Module css
+import './tabs.css';
+
+export const name = 'tabs';
+
+/**
+ * Components may need to run clean-up tasks if they are removed from DOM.
+ *
+ * @param {jQuery} $context - A piece of DOM
+ * @param {Object} settings - Pertinent settings
+ */
+// eslint-disable-next-line no-unused-vars
+export function disable($context, settings) {}
+
+/**
+ * Each component has a chance to run when its enable function is called. It is
+ * given a piece of DOM ($context) and a settings object. We destructure our
+ * component key off the settings object and provide an empty object fallback.
+ * Incoming settings override default settings via Object.assign().
+ *
+ * @param {jQuery} $context - A piece of DOM
+ * @param {Object} settings - Settings object
+ */
+export function enable($context) {
+  // Get relevant elements and collections
+  const $tabsWrapper = $('.tabs', $context);
+
+  const deactivatePanels = ($panels) => {
+    $panels.each(function panelsHidden() {
+      $(this).prop('hidden', true);
+    });
+  };
+
+  $tabsWrapper.each(function eachTabsGroup() {
+    const $this = $(this);
+    const $tablist = $('.tabs__list', $this);
+    const $tabs = $('.tabs__list-link', $this);
+    const $panels = $('.tabs__panels-item', $this);
+
+    // The tab switching function
+    const switchTab = (oldTab, newTab) => {
+      newTab.focus();
+      // Get id of panel controlled by newTab.
+      const newPanelId = $(newTab).attr('aria-controls');
+      // Make the active tab focusable by the user (Tab key) and add aria selected.
+      $(newTab).removeAttr('tabindex').attr('aria-selected', 'true');
+      $(oldTab).removeAttr('aria-selected').attr('tabindex', '-1');
+
+      // Hide all panels.
+      deactivatePanels($panels);
+      // Unhide new panel.
+      $(`#${newPanelId}`, $context).prop('hidden', false);
+    };
+
+    // Add semantics are remove user focusability for each tab
+    $tabs.each(function tabsInit(i, tab) {
+      $(tab).attr('tabindex', '-1');
```

Comment:
`tab` is already a jQuery object here. I think you're double-wrapping by `$(tab)`. Just send it in to your function as `$tab` and use it directly.